src/cljs/athens/components/components_util.cljs

```diff
@@ -0,0 +1,25 @@
+(ns athens.components.components-util
+  (:require
+    [athens.components.todo :as todo]
+    [athens.components.website-embeds :as website-embeds]))
+
+
+(defn default-component
+  [content uid]
+  ((constantly nil) uid)
```

Comment:
I see we're using `constantly` in many places. I don't understand what it's doing.

Reply:
Sorry for my Clojure noob-ness - I didn't know `_` existed. Fixed